From abba62fe1c22422bb84f12875fd3612b90ead434 Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Mon, 28 Feb 2011 08:14:30 -0600
Subject: [PATCH] editor: fix usage of nm_connection_update_secrets()

It takes the string:GValue hash of one settings' secrets, not
the hash of all settings.  Use it correctly.
---
 src/connection-editor/ce-page.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/connection-editor/ce-page.c b/src/connection-editor/ce-page.c
index 79730b0..b97e8c0 100644
--- a/src/connection-editor/ce-page.c
+++ b/src/connection-editor/ce-page.c
@@ -210,7 +210,7 @@ ce_page_complete_init (CEPage *self,
 	/* Update the connection with the new secrets */
 	if (nm_connection_update_secrets (self->connection,
 	                                  setting_name,
-	                                  secrets,
+	                                  setting_hash,
 	                                  &update_error)) {
 		/* Success */
 		emit_initialized (self, NULL);
-- 
1.7.4.1

